### ISSUE TITLE
Nested lists with 2+ levels

### ISSUE THREAD

**Issue by andreferreirav2** [notable/notable#165]
When creating lists with more than 2 levels, the 3rd level turns red and there is no actual change in the preview.

In order to create a 3rd level I need to add 3 tabs instead of 1.
Plus, there is a weird extra line spacing between the 1st and 2nd levels.

Ex:
![screenshot 2019-01-02 12 57 23](https://user-images.githubusercontent.com/2427597/50592893-f5e34280-0e8d-11e9-809f-2e4fc44e82c4.png)

Results in:
![screenshot 2019-01-02 12 53 44](https://user-images.githubusercontent.com/2427597/50592827-a7ce3f00-0e8d-11e9-9909-e74ea80dfc1f.png)

**Comment by fabiospampinato** [comment]
This will probably get fixed once we switch to Monaco #203

**Comment by fabiospampinato** [comment]
I think there are 2 issues here actually:

> When creating lists with more than 2 levels, the 3rd level turns red 

This is a problem with CodeMirror, and it will probably get fixed when we switch to Monaco #203.

> In order to create a 3rd level I need to add 3 tabs instead of 1.

This is a bug with showdown: https://github.com/showdownjs/showdown/issues/367

**Comment by fabiospampinato** [comment]
> When creating lists with more than 2 levels, the 3rd level turns red and there is no actual change in the preview.

We just switched to Monaco #203 and this is no longer a problem.

The bug about the rendering is still present though.

**Comment by kyldvs** [comment]
Running into the same issues with lists not being indented properly

![Screenshot (265)](https://user-images.githubusercontent.com/1830497/55193428-2d4b4d80-5164-11e9-9fa3-f2fc6198c284.png)

**Comment by Neko250** [comment]
If you use 4 spaces indentation instead of 2 spaces, it renders properly. :tada:

**Comment by fabiospampinato** [comment]
@Neko250 yeah, turned out our Markdown processor doesn't support 2-space indented lists. This will get fixed once we switch to marked #531

**Comment by fabiospampinato** [comment]
Fixed in v1.6.

But _ordered_ lists indented this way are not supported by CommonMark, for some reason that really makes no sense to me, I'll probably get this fixed too in a later release. https://github.com/commonmark/commonmark-spec/issues/594

**Comment by fabiospampinato** [comment]
Reopening as some patching of the Markdown compiler is required to make this work for ordered lists too.
